### PR TITLE
104 rewrite of manual for launch

### PR DESCRIPTION
**Pull request by wangzhao0217**
- update manual
- update

### CODE REVIEW

File: manual/index.html [hunk 1]
```diff
@@ -59,9 +59,8 @@ <h1>NPT Manual</h1>
...
-<p>The University of Leeds produces the NPT on behalf of Sustrans and Transport Scotland.</p>
+<p>Welcome to the Network Planning Tool (NPT) for Scotland, a cutting-edge web application designed for strategic cycle network planning. Developed by the University of Leeds, this tool is a collaborative effort with Sustrans and Transport Scotland. It extends the functionality of the <a href="https://www.pct.bike/">Propensity to Cycle Tool</a> for England and Wales, offering a detailed nationwide cycling potential analysis down to the street level (<a href="https://www.jtlu.org/index.php/jtlu/article/view/862">Lovelace et al. 2017</a>). As we officially launch Version 1, we invite users to explore its comprehensive features and contribute feedback for continuous improvement.</p>
```
Comment (Robinlovelace) [inline manual/index.html:62]: :+1:

File: manual/index.html [hunk 2]
```diff
@@ -72,7 +71,7 @@ <h3 id="essentials">NPT Essentials</h3>
 <p>The NPT is an open-source tool, meaning the source code is transparent and in the public domain for others to learn from and build on. The code underlying the project is available to be copied and improved by the community.</p>
 <p>The online interface can serve many needs, including gaining insight into the network of routes where cycling potential is highest and quantifying the benefits of cycling uptake. However, we recommend that intermediate and advanced users download data on zones, routes and route networks for in-house analysis.</p>
 <p>The NPT allows users to visualise results baseline data and how cycling would change under different scenarios. The journey purposes and scenarios are described in detail below.</p>
-<h2 id="access">Ways the access the NPT&nbsp;</h2>
+<h2 id="access">Accessing the NPT&nbsp;</h2>
```
Comment (Robinlovelace) [inline manual/index.html:74]: :+1: typo fix

File: manual/index.html [hunk 3]
```diff
@@ -103,7 +102,8 @@ <h4>macOS</h4>
     <li>Click install</li>
 </ol>
 <p>&nbsp;</p>
-<h2 id="user-interface">NPT User Interface</h2>
+<h2 id="user-interface">NPT User Interface Guide</h2>
+<p>The NPT's user interface is intuitive, featuring map controls for easy navigation and layer controls to customize data visibility. Whether you're interested in the baseline cycling data or exploring various "what-if" scenarios like "Go Dutch" or ebike adoption, the tool provides a flexible platform for in-depth analysis.</p>
```
Comment (Robinlovelace) [inline manual/index.html:106]: This reads well to me.

File: manual/index.html [hunk 4]
```diff
@@ -117,10 +117,10 @@ <h3 id="layer-controls">Layer Controls</h3>
 <p>&nbsp;</p>
 <p>&nbsp;</p>
 <h2 id="map-layers">Map Layers</h2>
-<p>The NPT allows you to view a broad range of data about cycling. This section describes each layer and its features.</p>
+<p>The NPT allows you to view a broad range of data about cycling by manipulating filters to showcase specific data points. This section describes each layer and its features.</p>
```
Comment (Robinlovelace) [inline manual/index.html:120]: :+1:

File: manual/index.html [hunk 5]
```diff
@@ -117,10 +117,10 @@ <h3 id="layer-controls">Layer Controls</h3>
...
 <h3 id="route-network">Route Network</h3>
-<p>The route network is the primary layer in the NPT and the only one that is on by default when the tool loads. The route network estimates the number of cyclists on each road. It is designed to emphasise roads with lots of cycling potential and so aid planners in deciding where cycling infrastructure is needed.</p>
-<p>The route network provides a range of options and filters to allow you to view different networks based on different journey purposes and assumptions.</p>
+<p>The Route Network is the primary layer in the NPT and the only one that is on by default when the tool loads. The route network estimates the number of cyclists on each road. It is designed to emphasise roads with lots of cycling potential and so aid planners in deciding where cycling infrastructure is needed.</p>
```
Comment (Robinlovelace) [inline manual/index.html:122]: :+1:

File: manual/index.html [hunk 6]
```diff
@@ -144,6 +144,20 @@ <h5>What&rsquo;s missing from the NPT</h5>
 <p>The NPT currently only considers direct journeys where the whole trip is by bicycle. It does not consider mixed-mode journeys such as cycling to the station and then taking a train to your final destination. This means that the NPT slightly underestimates cycling potential overall and may significantly underestimate cycling potential in specific places (such as around train stations).</p>
 </div>
 
+<div id="scenario">
+<h4>Scenarios</h4>
```
Comment (Robinlovelace) [inline manual/index.html:148]: :+1:

**Review by Robinlovelace — APPROVED** [review]

**Review by mem48 — APPROVED** [review]

**Pull request merged**